src/main/java/util/Utility.java

```diff
@@ -72,7 +72,10 @@ public static boolean writeFile(String fileName, String content, int issueCount)
                 writer.println(content);
                 writer.close();
 
-                long sizeAfterWrite = Files.size(Paths.get(fileName));
+                long sizeAfterWrite = 0;
+                if (issueCount > 0) {
```

Comment:
`Files.size` throws `IOException` if `issueCount` is 0? You mean if the file doesn't exist at the given path right? That doesn't really mean `issueCount` is 0, if that's what you mean.

You can put a check for whether the file exists in the first place (`Files.exists`) if that's the case, and then assign `sizeAfterWrite` accordingly.